back/app/serializers/web_api/v1/initiative_serializer.rb

```diff
@@ -36,9 +36,18 @@ class WebApi::V1::InitiativeSerializer < WebApi::V1::BaseSerializer
     can_moderate?(object, params)
   }
 
+  attribute :cosponsorships do |object, params|
+    name_service = UserDisplayNameService.new(AppConfiguration.instance, current_user(params))
+
+    object&.cosponsors_initiatives&.map do |ci|
+      { user_id: ci.user_id, name: name_service.display_name!(ci.user), status: ci.status }
```

Comment:
```suggestion
    object.cosponsors_initiatives.includes(:user).map do |ci|
      { user_id: ci.user_id, name: name_service.display_name!(ci.user), status: ci.status }
```
Without `includes`, there will be N+1. Can `object` be `nil`? If not, we don't need `&`.

Reply:
Good thinking. Thanks!